### PR TITLE
COMMERCE-4835 Can't add a SKU to a price list or sale price list or option value by searching by SKU

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/QueryPreProcessConfiguration.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 public interface QueryPreProcessConfiguration {
 
 	@Meta.AD(
-		deflt = "emailAddress|license|path|properties|screenName|tag|treePath|userName",
```
Comment (dsitu) [inline modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/QueryPreProcessConfiguration.java:36]: @riccardo-alberti I think we should sort this.

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/QueryPreProcessConfiguration.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
...
+		deflt = "emailAddress|license|path|properties|screenName|tag|treePath|userName|sku",
```
Comment (dsitu) [inline modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/QueryPreProcessConfiguration.java:36]: @riccardo-alberti I think we should sort this.

**Review by dsitu — APPROVED** [review]

**Review by marco-leo — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**